content/en/cloudcraft/advanced/find-id-using-api.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Find Cloud Account or Team ID using our API
+kind: guide
+---
+
+## Overview
+
+Currently, the Cloudcraft UI doesn't expose the ID of your AWS or Azure accounts or teams. However, you can still find these IDs using our API and a bit of manual work.
+
+## Prerequisites
+
+Before you begin, make sure you have the following:
+
+- A Cloudcraft user with the [Owner or Administrator role][1].
+- An active [Cloudcraft Pro subscription][2].
+- A Unix-like environment, such as Linux, macOS, or WSL on Windows with cURL and [jq][3] installed.
+- A basic understanding of the command-line interface.
+- A basic understanding of how to use APIs.
+
+You must also have at least one AWS or Azure account added to Cloudcraft.
```

Comment:
Is this really necessary?

Reply:
We have added it in previous articles of this nature, so I thought it made sense to include here too.